### ISSUE TITLE
`live_grep` not "fuzzy enough"

### ISSUE THREAD

**Issue by hood** [nvim-telescope/telescope.nvim#1854]
### Description

When searching for results with `Telescope live_grep`  search is not "fuzzy enough". I mean, if I'm looking for something like:
`TODO: Do this`
by searching
`TODO this`
I see no results appear.
I'm using `nvim-telescope/telescope-fzf-native.nvim` and `BurntSushi/ripgrep` together with Telescope. Same if I use `telescope-fzy-native.nvim` instead of fzf.

### Neovim version

```markdown
latest nightly (~0.8.something)
```

### Operating system and version

Arch Linux - 5.16.16-arch1-1

### checkhealth telescope

```markdown
telescope: require("telescope.health").check()
========================================================================
## Checking for required plugins
  - OK: plenary installed.
  - OK: nvim-treesitter installed.

## Checking external dependencies
  - OK: rg: found ripgrep 13.0.0
  - OK: fd: found fd 8.3.2

## ===== Installed extensions =====

## Telescope Extension: `fzf`
  - INFO: No healthcheck provided
```

### Steps to reproduce

invoking `Telescope live_grep` and searching for stuff is enough

### Expected behavior

Searching with `A C` should include results like `A B C` 

### Actual behavior

Searching with `A C` only includes results which include `A C` in its entirety

Example here:
![image](https://user-images.githubusercontent.com/56588510/163891624-a2c16e3c-c909-4cb3-8c47-23bf6d47b646.png)

I got "fuzzier" results when using `grep_string`:
![image](https://user-images.githubusercontent.com/56588510/163891595-e9af583d-c645-4a89-8bdd-ea6aa7d477f5.png)

But it still fails to find pretty darn close results:
![image](https://user-images.githubusercontent.com/56588510/163891849-e50ed21c-c1ac-422b-8dc2-eec5c1d5f9da.png)

### Minimal config

```Lua
vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvim/site]]
local package_root = '/tmp/nvim/site/pack'
local install_path = package_root .. '/packer/start/packer.nvim'
local function load_plugins()
  require('packer').startup {
    {
      'wbthomason/packer.nvim',
      {
        'nvim-telescope/telescope.nvim',
        requires = {
          'nvim-lua/plenary.nvim',
          { 'nvim-telescope/telescope-fzf-native.nvim', run = 'make' },
        },
      },
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. '/plugin/packer_compiled.lua',
      display = { non_interactive = true },
    },
  }
end
_G.load_config = function()
  require('telescope').setup()
  require('telescope').load_extension('fzf')
  -- ADD INIT.LUA SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
end
if vim.fn.isdirectory(install_path) == 0 then
  print("Installing Telescope and dependencies.")
  vim.fn.system { 'git', 'clone', '--depth=1', 'https://github.com/wbthomason/packer.nvim', install_path }
end
load_plugins()
require('packer').sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua load_config()]]

  require('telescope').setup {
    extensions = {
      fzf = {
        fuzzy = true,                    -- false will only do exact matching
        override_generic_sorter = true,  -- override the generic sorter
        override_file_sorter = true,     -- override the file sorter
        case_mode = "smart_case",        -- or "ignore_case" or "respect_case"
                                         -- the default case_mode is "smart_case"
      }
    }
  }

  require'telescope'.load_extension('fzf')
```

**Comment by Conni2461** [comment]
dup of #564

TLDR: from https://github.com/nvim-telescope/telescope.nvim/issues/564#issuecomment-786850776
> live_grep works a bit differently: it passes the typed text straight to rg (or friends) and shows the results.
>
> What you can do instead is :Telescope grep_string search='', which makes rg return all matches, and then you can use telescope to fuzzy filter. (Warning: this will be slow on large directories.)
